server/src/main/java/org/opensearch/cluster/routing/allocation/decider/DiskThresholdDecider.java

```diff
@@ -176,44 +167,9 @@ public static long sizeOfRelocatingShards(
     public Decision canAllocate(ShardRouting shardRouting, RoutingNode node, RoutingAllocation allocation) {
         ClusterInfo clusterInfo = allocation.clusterInfo();
 
-        /*
-         The following block enables allocation for remote shards within safeguard limits of the filecache.
-         */
+        // For this case WarmDiskThresholdDecider Decider will take decision
         if (REMOTE_CAPABLE.equals(getNodePool(node)) && REMOTE_CAPABLE.equals(getShardPool(shardRouting, allocation))) {
-            final double dataToFileCacheSizeRatio = fileCacheSettings.getRemoteDataRatio();
```

Comment:
Lets add a changelog for this stating that `Disk Threshold Deciders` works for warm shards - writable and searchable snapshots as well .